pkg/cel/ast/inspector.go

```diff
@@ -103,6 +103,8 @@ type Inspector struct {
 // knownFunctions contains the list of all CEL functions that are supported
 var knownFunctions = []string{
 	"random.seededString",
+	"base64.decode",
+	"base64.encode",
```

Comment:
Thanks a lot @Damy3 ! qq: do we need to manually hard code library functions as well? AFAIK this is not needed